presto-yarn-test/bin/run_on_docker.sh

```diff
@@ -33,8 +33,9 @@ function check_hive() {
 }
 
 function run_product_tests() {
+  MVN_VERSION=$(mvn -q -Dexec.executable="echo" -Dexec.args='${project.version}' --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
   compose run runner \
-    java -jar /workspace/target/presto-yarn-test-1.2.1-executable.jar \
+    java -jar /workspace/target/presto-yarn-test-$MVN_VERSION-executable.jar \
```

Comment:
it is not a version of maven but it is version of presto-yarn. So I would name it PRESTO_YARN_VERSION